lazy_tensor_core/lazy_tensor_core/csrc/tensor_ops.cpp

```diff
@@ -122,8 +124,18 @@ LazyTensor SmoothL1Loss(const LazyTensor& input, const LazyTensor& target,
     case ReductionMode::kNone:
       return elementwise_loss;
     case ReductionMode::kMean:
-      return LazyTensor::mean(elementwise_loss, all_dimensions, false,
-                              broadcasted_input.dtype());
+      //TODO(whc) SmoothL1Loss is not implemented by lazy TS backend,
```

Comment:
Nit: wong indention.

Reply:
this whole file seems wrong, i think my change fits in with the surroundings right?  prefer not to change the whole file in the same PR as other meaningful changes

Reply:
Oops, I mean `//TODO(whc)` => `// TODO(whc)`.

Reply:
ahh 👍